enterprise/frontend/src/embedding-sdk-bundle/components/public/MetabotQuestion/MetabotQuestion.tsx

```diff
@@ -43,10 +43,18 @@ const MetabotQuestionInner = ({
 
   const derivedLayout = useMemo(() => {
     return match(layout)
-      .with("auto", () =>
-        containerWidth <= MAX_MOBILE_CONTAINER_WIDTH ? "stacked" : "sidebar",
-      )
-      .otherwise((layout) => layout);
+      .with(P.union("stacked", "sidebar"), (layout) => layout)
+      .otherwise((layout) => {
+        if (layout !== "auto") {
```

Comment:
We default to `auto` if it's nullish already, so we can use this check for warnings.

Reply:
Oh boi, I thought this was done in CSS. But using data-attribute is kinda cool as we don't actually have to support `auto` in CSS.

I'm curious if people start to use container query in production yet. It's techincally possible that we do all this in CSS right? So code-wise we just need to pass along one of the 3 values to `data-attribute`

Reply:
@WiNloSt I was using container queries in CSS actually, but the CSS duplications needed are really messy, you'd have to update both the auto layout and the stacked/sidebar layout, so I ended up doing it in JS instead.